python-coverage-comment/README.md

```diff
@@ -0,0 +1,44 @@
+## Python code coverage comment
+
+Produces a pull request comment in markdown table format of the code coverage results
+
+## Example output
+
+| Name | Cover | Missing |
+| ---  | ----- | ------- |
+| **module//file.py** | 98% | 62 |
+| **module/file2.py** | 87% | 15,20 |
+
+### Total Coverage: 95%
+
+### Example use
+
+Example workflow could look something like this:
+
+```yaml
+
+jobs:
+  testing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run coverage
+        id: coverage
+        run: |
+          coverage json --data-file .coverage -o coverage.json
+          JSON=$(cat coverage.json)
+          echo "coverage=$JSON" >> $GITHUB_OUTPUT
```

Comment:
Should we also use job summaries? https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary

Reply:
Added